website/docs/r/google_project.html.markdown

```diff
@@ -62,8 +71,16 @@ The following arguments are supported:
     [below](#id-field) for more information about its behaviour.
 
 * `org_id` - (Optional) The numeric ID of the organization this project belongs to.
-    This is required if you are creating a new project.
-    Changing this forces a new project to be created.
+    Changing this forces a new project to be created.  If the `org_id`
+    is specified, but not the `folder_id`, then the project is created
+    at the top level. If both are specified, `org_id` is ignored.
```

Comment:
If you add the ConflictsWith I suggested earlier, you can update this comment to say that folder_id or org_id can be specified but not both.

Reply:
Done